script-injection.php

```diff
@@ -145,8 +204,7 @@ function do_category_js( $category ) {
   $event_key = "air_cookie_{$category_key}";
 
   ob_start(); ?>
-
-  if ( cc.allowedCategory( '<?php echo $category_key; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>' ) ) {
+  if ( CookieConsent.getCookie( 'categories' ).includes( '<?php echo $category_key; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>' ) ) {
```

Comment:
Why do we have to use this instead of [`CookieConsent.acceptedCategory()`](https://cookieconsent.orestbida.com/reference/api-reference.html#acceptedcategory)?

Reply:
Wasn't working correctly in 3.0.0, but I think that´s fixed in 3.0.1 now (PR is 3.0.0).

Reply:
Ah!